documentation/navitia.io/source/integration.rst

```diff
@@ -373,21 +376,134 @@ Response example for this request https://api.navitia.io/v1/coverage/fr-idf/phys
 Other examples
 
 * Network list
-  * https://api.navitia.io/v1/coverage/fr-idf/networks
+
+    * https://api.navitia.io/v1/coverage/fr-idf/networks
+
 * Physical mode list
-  * https://api.navitia.io/v1/coverage/fr-idf/physical_modes
+
+    * https://api.navitia.io/v1/coverage/fr-idf/physical_modes
+
 * Line list
-  * https://api.navitia.io/v1/coverage/fr-idf/lines
+
+    * https://api.navitia.io/v1/coverage/fr-idf/lines
+
 * Line list for one mode
-  * https://api.navitia.io/v1/coverage/fr-idf/physical_modes/physical_mode:Metro/lines
 
+    * https://api.navitia.io/v1/coverage/fr-idf/physical_modes/physical_mode:Metro/lines
+
+
+pt_objects (/pt_objects)
```

Comment:
I'm not sure, but maybe it's better for the title to be:
Public Transport objects autocomplete (/pt_objects)